buildSrc/src/main/java/dependencies/Dependencies.kt

```diff
@@ -67,7 +67,7 @@ object Dependencies {
     const val slf4j_api = "org.slf4j:slf4j-api:2.0.6"
     const val slf4j_timber = "com.arcao:slf4j-timber:3.1@aar"
     const val emoji_java = "com.vdurmont:emoji-java:5.1.1"
-    const val json_schema_validator = "com.networknt:json-schema-validator:1.0.75"
+    const val json_schema_validator = "com.networknt:json-schema-validator:1.0.72"
```

Comment:
I would prefer to have that comment about not upgrading the dependency here like in the case of `commons_io` for example. It's well visible and when I go through the list of dependencies at the beginning of every release cycle it's clear that we shouldn't touch it.

Reply:
Right, I see what you mean. I guess my worry was that another dependency could cause the same problem by depending on a newer Jackson (given it's such a popular library). I do see that it seems strange to downgrade `json_schema_validator` **AND** lock Jackson though. I'll remove this version change so that we're just locking Jackon.